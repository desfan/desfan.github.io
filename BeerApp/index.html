<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8" />
	<!--[if lt IE 9]><script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js"></script><![endif]-->
	<title></title>
	<meta name="keywords" content="" />
	<meta name="description" content="" />
	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4" crossorigin="anonymous">
	<link href="css/style.css" rel="stylesheet">
</head>

<body>
	<header>
        <div class="container">
            <div class="row">

                <!--Шапка-->
                <div class="col-xl-6 d-flex">
                    <div class="col-md-7">
                        <div class="wrap"><p>name</p></div>                                            
                    </div>
                    <div class="col-md-2">
                        <div class="wrap"><p>max/min</p></div>
                    </div>
                    <div class="col-md-2">
                        <div class="wrap"><p>price</p></div>
                    </div>
                    <div class="col-md-1">
                        <div class="wrap"><p>vol</p></div>
                    </div>
                </div>

                <div class="col-xl-6 d-flex">
                        <div class="col-md-7">
                            <div class="wrap"><p>name</p></div>                                            
                        </div>
                        <div class="col-md-2">
                            <div class="wrap"><p>max/min</p></div>
                        </div>
                        <div class="col-md-2">
                            <div class="wrap"><p>price</p></div>
                        </div>
                        <div class="col-md-1">
                            <div class="wrap"><p>vol</p></div>
                        </div>
                </div>

                <!--Данные 1-я строка-->
                <div class="col-xl-6 d-flex">
                    <div class="col-md-7">
                        <span class="name">varvar ipa nema</span>                                            
                    </div>
                    <div class="col-md-2">
                        <span class="max">110</span><span class="slash"> / </span><span class="min">45</span>
                    </div>
                    <div class="col-md-2">
                        <span class="price">102<sup>40</sup></span>
                    </div>
                        <div class="col-md-1">
                        <span class="vol">0.4</span>
                    </div>
                </div>

                <div class="col-xl-6 d-flex">
                    <div class="col-md-7">
                        <span class="name">fdb happy stout</span>                                            
                    </div>
                    <div class="col-md-2">
                        <span class="max">102</span><span class="slash"> / </span><span class="min">45</span>
                    </div>
                    <div class="col-md-2">
                        <span class="price">98<sup>90</sup></span>
                    </div>
                        <div class="col-md-1">
                        <span class="vol">0.4</span>
                    </div>
                </div>

                <!--Данные 2-я строка-->
                <div class="col-xl-6 d-flex">
                    <div class="col-md-7">
                        <span class="name">varvar hoppy lager</span>                                            
                    </div>
                    <div class="col-md-2">
                        <span class="max">110</span><span class="slash"> / </span><span class="min">45</span>
                    </div>
                    <div class="col-md-2">
                        <span class="price">92<sup>80</sup></span>
                    </div>
                    <div class="col-md-1">
                        <span class="vol">0.4</span>
                    </div>
                </div>

                <div class="col-xl-6 d-flex">
                    <div class="col-md-7">
                        <span class="name">hb weissbier</span>                                            
                    </div>
                    <div class="col-md-2">
                        <span class="max">100</span><span class="slash"> / </span><span class="min">45</span>
                    </div>
                    <div class="col-md-2">
                        <span class="price">97<sup>10</sup></span>
                    </div>
                    <div class="col-md-1">
                        <span class="vol">0.4</span>
                    </div>
                </div>

                <!--Данные 3-я строка-->
                <div class="col-xl-6 d-flex">
                    <div class="col-md-7">
                        <span class="name">ципа біла</span>                                            
                    </div>
                    <div class="col-md-2">
                        <span class="max">100</span><span class="slash"> / </span><span class="min">45</span>
                    </div>
                    <div class="col-md-2">
                        <span class="price">91<sup>70</sup></span>
                    </div>
                    <div class="col-md-1">
                        <span class="vol">0.4</span>
                    </div>
                </div>
    
                <div class="col-xl-6 d-flex">
                    <div class="col-md-7">
                        <span class="name">fdb goze</span>                                            
                    </div>
                    <div class="col-md-2">
                        <span class="max">100</span><span class="slash"> / </span><span class="min">45</span>
                    </div>
                    <div class="col-md-2">
                        <span class="price">91<sup>70</sup></span>
                    </div>
                    <div class="col-md-1">
                        <span class="vol">0.4</span>
                    </div>
                </div>

                <!--Данные 4-я строка-->
                <div class="col-xl-6 d-flex">
                        <div class="col-md-7">
                            <span class="name">collider mozambique</span>                                            
                        </div>
                        <div class="col-md-2">
                            <span class="max">100</span><span class="slash"> / </span><span class="min">45</span>
                        </div>
                        <div class="col-md-2">
                            <span class="price">91<sup>70</sup></span>
                        </div>
                        <div class="col-md-1">
                            <span class="vol">0.4</span>
                        </div>
                    </div>
        
                    <div class="col-xl-6 d-flex">
                        <div class="col-md-7">
                            <span class="name">super biter baladin</span>                                            
                        </div>
                        <div class="col-md-2">
                            <span class="max">100</span><span class="slash"> / </span><span class="min">45</span>
                        </div>
                        <div class="col-md-2">
                            <span class="price">93<sup>90</sup></span>
                        </div>
                        <div class="col-md-1">
                            <span class="vol">0.33</span>
                        </div>
                    </div>

                <!--Данные 5-я строка-->
                <div class="col-xl-6 d-flex">
                    <div class="col-md-7">
                        <span class="name">magners cider</span>                                            
                    </div>
                    <div class="col-md-2">
                        <span class="max">100</span><span class="slash"> / </span><span class="min">45</span>
                    </div>
                    <div class="col-md-2">
                        <span class="price">91<sup>70</sup></span>
                    </div>
                    <div class="col-md-1">
                        <span class="vol">0.4</span>
                    </div>
                </div>
        
                    <div class="col-xl-6 d-flex">
                        <div class="col-md-7">
                            <span class="name">fdb white ipa</span>                                            
                        </div>
                        <div class="col-md-2">
                            <span class="max">100</span><span class="slash"> / </span><span class="min">45</span>
                        </div>
                        <div class="col-md-2">
                            <span class="price">93<sup>50</sup></span>
                        </div>
                        <div class="col-md-1">
                            <span class="vol">0.4</span>
                        </div>
                    </div>


                


            </div>
        </div> 
        
        
           
        
            
    </header>
    
                    
    
</body>
</html>