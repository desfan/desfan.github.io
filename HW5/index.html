<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Restaurant</title>
	<link rel="stylesheet" href="css/style.css">
	<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400&amp;subset=cyrillic" rel="stylesheet"> 
</head>

<body>	
	<div class="wrapper">
		<div class="nav">
			<h2>
				Fountain
			</h2>
			<div class="nav_button">
				<img src="img/nav_button.png" alt="">
			</div>		
		</div>

		<div class="poster">			
			<h1>
				OUR FOOD <strong>RESTORES, REPLENISHES,</strong> AND <strong>REVIVES.</strong>
			</h1>
		</div>

		<div class="content">		
			<div class="content__img">
				<img src="img/About_Image.png" alt="">
			</div>
			
			<div class="text__block">
				<h3>
					About
				</h3>
				<p>
					Tacos squid enamel pin pok pok wayfarers tote bag mustache, air plant meditation la croix blue bottle celiac everyday carry tousled hexagon. Trust fund stumptown green juice, migas ennui hella air plant. Bitters forage vexillologist wolf tousled, DIY gluten-free fanny pack.
				</p>
				<p>
					<strong>Petar Petrović</strong> Executive Chef & Owner
				</p>				
			</div>		
		</div>

		<div class="content">			
			<div class="text__block">
				<h3>
					Why organic ?!
				</h3>
				<p>
					Tacos squid enamel pin pok pok wayfarers tote bag mustache, air plant meditation la croix blue bottle celiac everyday carry tousled hexagon. Trust fund stumptown green juice, migas ennui hella air plant. Bitters forage vexillologist wolf tousled, DIY gluten-free fanny pack.
				</p>
				<p>
					<strong>Marko Marković</strong> Master Chef
				</p>				
			</div>
			<div class="content__img">
				<img src="img/Organic_Image.png" alt="">
			</div>
		</div>

		<div class="content">		
			<div class="content__img">
				<img src="img/Traditional_Image.png" alt="">
			</div>
			
			<div class="text__block">
				<h3>
					Traditional
				</h3>
				<p>
					Tacos squid enamel pin pok pok wayfarers tote bag mustache, air plant meditation la croix blue bottle celiac everyday carry tousled hexagon. Trust fund stumptown green juice, migas ennui hella air plant. Bitters forage vexillologist wolf tousled, DIY gluten-free fanny pack.
				</p>
				<p>
					<strong>Zoran Zoranić</strong> Chef Baker
				</p>				
			</div>		
		</div>

		<div class="meat">		
			<div class="fon__meat">
				<h3 class="text__meat">
					Meat from farms
				</h3>
				<p class="text__meat">
					Tacos squid enamel pin pok pok wayfarers tote bag mustache, air plant meditation la croix blue bottle celiac everyday carry tousled hexagon. Trust fund stumptown green juice, migas ennui hella air plant. Bitters forage vexillologist wolf tousled, DIY gluten-free fanny pack.
				</p>
			</div>		
		</div>

		<div class="menu">
			<div class="container">
				<h4>Menu</h4>
				<p>Our menu rotates within the season.</p>
				<p>Here is a list of dishes you may see this week.</p>
				<hr>
				<p>
					<strong>Hipster Ipsum</strong> - Keytar four loko single-origin coffee jianbing tbh, cred tofu bitters flannel pour-over cold-pressed raw denim banh mi mlkshk. 
				</p>
				<p>
					<strong>Hipster Ipsum</strong> - Keytar four loko single-origin coffee jianbing tbh, cred tofu 
				</p>
				<p>
					<strong>Hipster Ipsum</strong> - Keytar four loko single-origin coffee jianbing tbh, cred tofu bitters flannel pour-over.
				</p>
				<p>
					<strong>Hipster Ipsum</strong> - Keytar four loko single-origin coffee jianbing tbh 
				</p>
				<p>
					<strong>Hipster Ipsum</strong> - Keytar four loko single-origin coffee jianbing tbh, cred tofu bitters flannel. 
				</p>
				<p>* * *</p>
				<p>
					<strong>Hipster Ipsum</strong> - Keytar four loko single-origin coffee jianbing tbh, cred tofu bitters flannel pour-over cold-pressed raw denim banh mi mlkshk. 
				</p>
				<p>
					<strong>Hipster Ipsum</strong> - Keytar four loko single-origin coffee jianbing tbh, cred tofu bitters flannel pour-over. 
				</p>
				<p>
					<strong>Hipster Ipsum</strong> - Keytar four loko single-origin coffee jianbing tbh. 
				</p>
				<p>
					<strong>Hipster Ipsum</strong> - Keytar four loko single-origin coffee jianbing tbh, cred tofu bitters flannel. 
				</p>
				<p>* * *</p>
				<p>
					<strong>Hipster Ipsum</strong> - Keytar four loko single-origin coffee jianbing tbh, cred tofu bitters flannel pour-over cold-pressed raw denim banh mi mlkshk. 
				</p>
				<p>
					<strong>Hipster Ipsum</strong> - Keytar four loko single-origin coffee jianbing tbh, cred tofu. 
				</p>
				<p>
					<strong>Hipster Ipsum</strong> - Keytar four loko single-origin coffee jianbing tbh, cred tofu bitters flannel pour-over. 
				</p>
				<hr>			
			
				</div>
		</div>	
		

			<div class="img__block">
				<img src="img/Meat_Menu.png" alt="">
			</div>

		<div class="container">
				<h4>
					Reservation
				</h4>
				<p>
					We'd love for you to dine with us. Reservations up to 30 days in advance.  
				</p>

				<div class="button">
					Book a table
				</div>
		</div>	

		<div class="tea">		
			<div class="fon__meat">
				<h3 class="text__meat">
					Tea after a meal
				</h3>
				<p class="text__meat">
					Tacos squid enamel pin pok pok wayfarers tote bag mustache, air plant meditation la croix blue bottle celiac everyday carry tousled hexagon. Trust fund stumptown green juice, migas ennui hella air plant.
				</p>
			</div>		
		</div>

		<div class="schedule">
				<h5>Dine with us</h5>
				<hr>
				<p>M-Tu</p>
				<h5>Closed</h5>
				<p>W-Th</p>
				<h5>6:00 - 10:00 pm</h5>
				<p>F-Su</p>
				<h5>5:30 - 10:00 pm</h5>
				<hr>
				<h6>Street N.N.</h6>
				<h6>Montenegro</h6>
				<h6>(+382) 55 555 555</h6>
				<h6>info@fountanrestaurant.me</h6>

				<form class="subscribe" action="#" method="post">
                    <h4>
                        JOIN US FOR SPECIAL EVENT ANNOUNCEMENTS
                        <br> AND BEHIND-THE-SCENES STORIES.
                    </h4>
                    <div class="form">
                        <input type="email" placeholder="Enter your email">
                        
                        <button>Subscribe</button>
                    </div>
                </form>
		</div>


		</div>


		<div class="footer">
			<p>©2017 Fountan Restaurant</p> 
		</div>	
		

</body>
</html>