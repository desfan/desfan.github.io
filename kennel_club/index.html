<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8" />
	<!--[if lt IE 9]><script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js"></script><![endif]-->
	<title>Дрессировка собак Днепр, кинолог,зоопсихолог,  зоогостиница (передержка собак)</title>
	<meta name="keywords" content="" />
	<meta name="description" content="" />
	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
	<link href="css/style.css" rel="stylesheet">
</head>

<body>

<!-- <div class="wrapper"> -->

	<header>
		<div class="header-container">
			<div class="logo">
				<img src="img/logo.png" alt="LogoK9">
			</div>
				<input type="checkbox" id="menu-checkbox">
				<nav role="navigation">
					<label for="menu-checkbox" class="toggle-button"
						data-open="МЕНЮ" data-close="МЕНЮ" onclick></label>
					<ul class="main-menu">
						<li><a href="#">Главная</a></li>
						<li><a href="#">О нас</a></li>
						<li><a href="#">Школа К9</a></li>
						<li><a href="#">Догхостел</a></li>
						<li><a href="#">Контакты</a></li>
					</ul>
				</nav>
		</div>
		<div class="top-container">
			<div><h2>Ваше с питомцем</h2></div>
			<div><h2>взаимопонимание -</h2></div>
			<div><h2>наш личный успех!</h2></div>
			<div><h3>Мы - семья кинологов, основателей школы “К9 Днепр” <br>и гостиницы для собак “Догхостел Дача”</h3></div>

			<div class="social">
				<div><img src="img/fb.png"></div>
				<div><img src="img/inst.png"></div>
				<div><img src="img/youtube.png"></div>
			</div>
		</div>		

		

		
	</header><!-- .header-->



	<footer class="footer">
		<strong>Footer:</strong> Mus elit Morbi mus enim lacus at quis Nam eget morbi. Et semper urna urna non at cursus dolor vestibulum neque enim. Tellus interdum at laoreet laoreet lacinia lacinia sed Quisque justo quis. Hendrerit scelerisque lorem elit orci tempor tincidunt enim Phasellus dignissim tincidunt. Nunc vel et Sed nisl Vestibulum odio montes Aliquam volutpat pellentesque. Ut pede sagittis et quis nunc gravida porttitor ligula.
	</footer><!-- .footer -->

<!-- </div> --><!-- .wrapper -->
 
<script src="https://use.fontawesome.com/b5b0df6556.js"></script>
</body>
</html>