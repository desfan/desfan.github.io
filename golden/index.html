<!doctype html>
<html lang="en">
  <head>
    <!-- Required meta tags -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

    <!-- Bootstrap CSS -->
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700|Roboto+Slab" rel="stylesheet"> 
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.9/css/all.css" integrity="sha384-5SOiIsAziJl6AWe0HWRKTXlfcSHKmYV4RBF18PPJ173Kzn7jzMyFuTtk8JA7QQG1" crossorigin="anonymous">
    <link rel="stylesheet" href="css/main.css">

    <title>Golden</title>
  </head>
  <body>

    <!-- First Screen -->

    <header id="header" class="header">
      <div class="container">
        <div class="row">
          <div class="col-ml-4 col-sm-3 m-auto">
            <img class="logo" src="img/Logo.png" alt="Logo_Golden">
          </div>
          <div class="col-ml-8 col-sm-9 m-auto">


            



            <nav>
              <ul class="menu">
                <li class="menu__item"><a href="#header">Home</a></li>
                <li class="menu__item"><a href="#services">Services</a></li>
                <li class="menu__item"><a href="#portfolio">Portfolio</a></li>
                <li class="menu__item"><a href="#about">About</a></li>
                <li class="menu__item"><a href="#contact">Contact</a></li>          
              </ul>
            </nav>
          </div>
          <div class="row m-auto">
            <div class="col-lg-12">
              <h2>
                Welcome To Our Studio!
              </h2>
              <h1>
                Itâ€™s nice to Meet you
              </h1>
            </div>
            <div class="col-lg-12 d-flex justify-content-center">
              <div class="btn yellow">
                Tell me more
              </div>
            </div>
          </div>
        </div>
      </div>
    </header>

    <!-- Services Screen -->

    <section id="services" class="services">
      <div class="container">

        <div class="row">
          <div class="col-lg-12">
            <h3>Services</h3>
            <h5>Proin iaculis purus consequat sem cure. </h5>
          </div>
        </div>

        <div class="row">
          <div class="col-lg-4 d-flex justify-content-center">
            <div class="thumbnail">
              <div class="services__icon m-auto">
                <a href="#"><i class="fas fa-shopping-basket"></i></a>
              </div>              
              <div class="caption">
                <h4>E-Commerce</h4>
                <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
              </div>
            </div>
          </div>

          <div class="col-lg-4 d-flex justify-content-center">
            <div class="thumbnail">
              <div class="services__icon m-auto">
                <a href="#"><i class="fas fa-laptop"></i></a>
              </div>
              <div class="caption">
                <h4>Responsive Web</h4>
                <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
              </div>
            </div>
          </div>

          <div class="col-lg-4 d-flex justify-content-center">
            <div class="thumbnail">
              <div class="services__icon m-auto">
                <a href="#"><i class="fas fa-lock"></i></a>
              </div>
              <div class="caption">
                <h4>Web Security</h4>
                <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
              </div>
            </div>
          </div>

        </div>
      </div>
    </section>

    <!-- Portfolio Screen -->

    <section id="portfolio" class="portfolio">
      <div class="container">
        <div class="row">
          <div class="col-lg-12">
            <h3>Our Portfolio</h3>
            <h5>Proin iaculis purus consequat sem cure. </h5>
          </div>
        </div>

        <div class="row">
          <div class="col-sm-6 col-lg-4 d-flex justify-content-center">
              <div id="card" class="thumbnail">
                <a target="_blank" href="#">
                  <img src="img/img1.jpg" alt="img1.jpg">
                    <div class="caption">
                      <h4>Ebony & Ivory</h4>
                      <h6>Branding</h6>
                    </div>
                </a>
              </div>
          </div>

          <div class="col-sm-6 col-lg-4 d-flex justify-content-center">
            <div id="card" class="thumbnail">
                <a target="_blank" href="#">
                  <img src="img/img2.jpg" alt="img2.jpg">
                    <div class="caption">
                      <h4>Smart Stationary</h4>
                      <h6>Print Design</h6>
                    </div>
                </a>
              </div>
          </div>

          <div class="col-sm-6 col-lg-4 d-flex justify-content-center">
            <div id="card" class="thumbnail">
              <a target="_blank" href="#">
                <img src="img/img3.jpg" alt="img3.jpg">
                  <div class="caption">
                    <h4>Clever Poster</h4>
                    <h6>Print Design</h6>
                  </div>
                </a>
            </div>
          </div>

          <div class="col-sm-6 col-lg-4 d-flex justify-content-center">
            <div id="card" class="thumbnail">
              <a target="_blank" href="#">
                <img src="img/img4.jpg" alt="img4.jpg">
                  <div class="caption">
                    <h4>Vinyl Records</h4>
                    <h6>Product Mock-Up</h6>
                  </div>
                </a>
            </div>
          </div>

          <div class="col-sm-6 col-lg-4 d-flex justify-content-center">
            <div id="card" class="thumbnail">
              <a target="_blank" href="#">
                <img src="img/img5.jpg" alt="img5.jpg">
                  <div class="caption">
                    <h4>Threehouse Template</h4>
                    <h6>Web Design</h6>
                  </div>
                </a>
            </div>
          </div>

          <div class="col-sm-6 col-lg-4 d-flex justify-content-center">
            <div id="card" class="thumbnail">
              <a target="_blank" href="#">
                <img src="img/img6.jpg" alt="img6.jpg">
                  <div class="caption">
                    <h4>Burned Logo</h4>
                    <h6>Branding</h6>
                  </div>
                </a>
            </div>
          </div>
        </div>        
      </div>
    </section>

    <!-- About Screen -->

    <section id="about" class="about">
      <div class="container">
        <div class="row">
          <div class="col-lg-12">
            <h3>About US</h3>
            <h5>Proin iaculis purus consequat sem cure.</h5>
          </div>
        </div>

        <div class="container">
        <div class="row m-auto">
          <div class="col-sm-12 col-lg-5 d-flex justify-content-end">
            <div class="wrap">
              <h4 class="about__text-right">
                  JULY 2010<br>
                  Our Humble Beginnings
                </h4>
                <p class="about__text-right">
                  Proin iaculis purus consequat sem curedigni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.
                </p>
            </div>
          </div>
          <div class="col-sm-12 col-lg-2">
            <div class="wrap">
              <img src="img/img7.png" class="m-auto">
            </div>
            <div class="line m-auto">
            
            </div>
          </div>
          <div class="col-lg-5">
                      
          </div>
        </div>
        </div>

        <div class="row">
          <div class="col-lg-5 d-flex justify-content-end">
            
          </div>
          <div class="col-lg-2">
            <div class="wrap">
              <img src="img/img8.png">
            </div>
            <div class="line m-auto">
            
            </div>
          </div>
          <div class="col-lg-5">
            <div class="wrap">
              <h4 class="about__text-left">
                  JANUARY 2011<br>
                  Facing Startup Battles
                </h4>
                <p class="about__text-left">
                  Proin iaculis purus consequat sem cure 
                  digni ssim. Donec porttitora entum suscipit 
                  aenean rhoncus posuere odio in tincidunt. Proin 
                  iaculis purus consequat sem cure digni 
                  ssim. Donec porttitora entum suscipit aenean 
                  rhoncus posuere odio in tincidunt.
                </p>
            </div>            
          </div>
        </div>

        <div class="row">
          <div class="col-lg-5 d-flex justify-content-end">
            <div class="wrap">
              <h4 class="about__text-right">
                  DECEMBER 2012<br>
                  Enter The Dark Days
                </h4>
                <p class="about__text-right">
                  Proin iaculis purus consequat sem curedigni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni.
                </p>
            </div>
          </div>
          <div class="col-lg-2">
            <div class="wrap">
              <img src="img/img9.png">
            </div>
            <div class="line m-auto">
            
            </div>
          </div>
          <div class="col-lg-5">
                      
          </div>
        </div>

        <div class="row">
          <div class="col-lg-5 d-flex justify-content-end">
            
          </div>
          <div class="col-lg-2">
            <div class="wrap">
              <img src="img/img10.png">
            </div>
            <div class="line m-auto">
            
            </div>
          </div>
          <div class="col-lg-5">
            <div class="wrap">
              <h4 class="about__text-left">
                  FEBRUARY 2014<br>
                  Our Triumph
                </h4>
                <p class="about__text-left">
                  Proin iaculis purus consequat sem cure 
                  digni ssim. Donec porttitora entum suscipit 
                  aenean rhoncus posuere odio in tincidunt. Proin 
                  iaculis purus consequat sem cure digni 
                  ssim. Donec porttitora entum suscipit aenean.
                </p>
            </div>            
          </div>
        </div>

        <div class="row">
          <div class="col-lg-5 d-flex justify-content-end">
            
          </div>
          <div class="col-lg-2">
            <div class="wrap">
              <img src="img/img11.png">
            </div>
          </div>
          <div class="col-lg-5">
                    
          </div>
        </div>
    </section>

    <!-- Amazing Screen -->

    <section id="amazing" class="amazing">
      <div class="container">
        <div class="row">
          <div class="col-lg-12">
            <h3>Our amazing team</h3>
            <h5>Proin iaculis purus consequat sem cure.</h5>
          </div>
        </div>
      </div>
        
      <div class="container">
        <div class="row">

          <div class="col-lg-4 d-flex justify-content-center">
            <div class="thumbnail">
              <img src="img/Marketer.png">
              <div class="caption">
                <h4>Kimberly Thompson</h4>
                <p>Marketer</p>
                  <ul class="social d-flex justify-content-center">
                    <li><a href="#"><i class="fab fa-twitter" aria-hidden="true"></i></a></li>
                    <li><a href="#"><i class="fab fa-facebook-f" aria-hidden="true"></i></a></li>
                    <li><a href="#"><i class="fab fa-pinterest-p" aria-hidden="true"></i></a></li>
                  </ul>
              </div>
            </div>
          </div>

          <div class="col-lg-4 d-flex justify-content-center">
            <div class="thumbnail">
              <img src="img/Marketer.png">
              <div class="caption">
                <h4>Rico Massimo</h4>
                <p>Coder</p>
                  <ul class="social d-flex justify-content-center">
                    <li><a href="#"><i class="fab fa-twitter" aria-hidden="true"></i></a></li>
                    <li><a href="#"><i class="fab fa-facebook-f" aria-hidden="true"></i></a></li>
                    <li><a href="#"><i class="fab fa-pinterest-p" aria-hidden="true"></i></a></li>
                  </ul>
              </div>
            </div>
          </div>

          <div class="col-lg-4 d-flex justify-content-center">
            <div class="thumbnail">
              <img src="img/Marketer.png">
              <div class="caption">
                <h4>Uku Mason</h4>
                <p>Graphic Designer</p>
                  <ul class="social d-flex justify-content-center">
                    <li><a href="#"><i class="fab fa-twitter" aria-hidden="true"></i></a></li>
                    <li><a href="#"><i class="fab fa-facebook-f" aria-hidden="true"></i></a></li>
                    <li><a href="#"><i class="fab fa-pinterest-p" aria-hidden="true"></i></a></li>
                  </ul>
              </div>
            </div>
          </div>

        </div>
      </div>
    </section>

    <!-- CMS Screen -->

    <section id="cms" class="cms ">
      <div class="container d-flex justify-content-center">
          <div class="cms__content">
            <img src="img/Envato.png" alt="">
            <img src="img/WordPress.png" alt="">
            <img src="img/Tuts.png" alt="">
            <img src="img/Microlancer.png" alt="">
          </div>
      </div>
    </section>

    <!-- Contact Screen -->

    <section id="contact" class="contact">
      <div class="container">
        <div class="row">
          <div class="col-lg-12">
            <h3>Contact US</h3>
            <h5>Proin iaculis purus consequat sem cure.</h5>
          </div>
        </div>


        <form>
          <div class="row">
            <div class="col-lg-6">
              <div class="form-group">
                <input type="text" class="form-control" id="validationCustom01" placeholder="Your name" required>   
              </div>

            <div class="form-group">
              <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="Your e-mail" required>
            </div>

            <div class="form-group">
              <input type="text" class="form-control" id="validationCustom01" placeholder="Subject" required>
            </div>
          </div>

            <div class="col-lg-6">
              <div class="form-group">
                <textarea class="form-control" id="exampleFormControlTextarea1" placeholder="Your message" rows="6"></textarea>
              </div>
            </div>
          </div>

          <div class="row">
            <div class="col-lg-12 d-flex justify-content-center">
              <button type="submit" class="btn yellow">Send message</button>
            </div>
          </div>      
        </form>
      </div>
    </section>

    <!-- Footer -->

    <footer id="footer" class="footer">
      <div class="container">
        <div class="row">
          <div class="col-ml-4 col-lg-4 d-flex justify-content-end">
            <h6>Â© Copyright 2014 FreebiesXpress.com</h6>
          </div>
          <div class="col-ml-4 col-lg-4">
            <ul class="d-flex justify-content-center">
                <li class="social"><a href="#"><i class="fab fa-twitter" aria-hidden="true"></i></a></li>
                <li class="social"><a href="#"><i class="fab fa-facebook-f" aria-hidden="true"></i></a></li>
                <li class="social"><a href="#"><i class="fab fa-pinterest-p" aria-hidden="true"></i></a></li>
                <li class="social"><a href="#"><i class="fab fa-google-plus-g" aria-hidden="true"></i></a></li>
            </ul>
          </div>
          <div class="col-ml-4 col-lg-4">
                      
          </div>
        </div>
      </div>
    </footer> 

    <!-- Optional JavaScript -->
    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
  </body>
</html>